requirements.txt

```diff
@@ -9,5 +9,6 @@ Pygments~=2.1
 PyPrint~=0.2.6
 requests~=2.12
 setuptools>=19.0
-testfixtures~=4.14.3
 unidiff~=0.5.2
+testfixtures~=5.3.1
```

Comment:
please don't change the position of `testfixtures`, this does not adhere now alphabetical order ;)